data/low_income_thresholds.json

```diff
@@ -154,6 +154,21 @@
       }
     }
   },
+  "GA": {
```

Comment:
GA doesn't have any income-qualified programs but the schema test requires a default is filled out. This just uses the federal poverty guidelines